scripts/release-tools/downstream/y-stream-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+previous_release=$1
+release=$2
+
+get_supported_versions() {
+    supported_json="$(curl https://access.redhat.com/product-life-cycles/api/v1/products?name=Red%20Hat%20Advanced%20Cluster%20Security%20for%20Kubernetes)"
```

Comment:
```suggestion
    supported_json="$(curl -fsSL https://access.redhat.com/product-life-cycles/api/v1/products?name=Red%20Hat%20Advanced%20Cluster%20Security%20for%20Kubernetes)"
```

https://explainshell.com/explain?cmd=curl+-fsSL+example.org